plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2719,6 +2731,17 @@ private KVMPhysicalDisk getPhysicalDiskPrimaryStore(PrimaryDataStoreTO primaryDa
         return storagePool.getPhysicalDisk(data.getPath());
     }
 
+    protected void setDiskIoDriver(DiskDef disk) {
+        /*
+         * IO Driver works for
+         * - Qemu >= 5.0
+         * - Libvirt >= 6.3.0
+         */
```

Comment:
Instead of a comment, could we turn it into a javadoc for the method?

Reply:
Done :)